03Lesson/01Project/index.html

```diff
@@ -1,9 +1,64 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <meta charset="UTF-8">
+  <title>Intel</title>
+  <link rel="stylesheet" href="css/style.css" />
 </head>
+
 <body>
+  <header>
+    <h1>
+      Intel
+    </h1>
+    <nav class="nav-link">
+      <span>i7 Procressor</span>
+      <span>Solid-State Drives</span>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h2>Intel® Core™ i7</h2>
+      <p>The 6th Generation Intel® Core™ i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even the toughest of tasks and games.</p>

```

Comment:
I would put all block element tags on their own line and indent the content and child elements.